docs/index.rst

```diff
@@ -12,12 +12,124 @@ COSMOS Academy
    :alt: cosmos-academy-logo
    :align: center
 
+Welcome to the Cosmos Academy! This page will orient you around the Tendermint/Cosmos tech stack. It is roughly organized from beginner => expert with higher priority to content most relevant to many users.
 
-Contents
+Introduction
```

Comment:
Here is how I would organized the sections:

1. Introduction
2. Tendermint & Cosmos
3. App Development
4. Testnets
5. Glossary
6. Concepts
7. Research
8. Media (Include Blog and Videos)
9. Community
10. External Resources

I would remove "ReadTheDocs" section and link to the Docs direclty in the relevant sections.

Reply:
I like it